utils/symlink-dependencies/symlink-dependencies.js

```diff
@@ -77,7 +77,10 @@ function symlinkDependencies(packages, packageGraph, tracker) {
         });
 
         // create package symlink
-        chain = chain.then(() => createSymlink(dependencyNode.location, targetDirectory, "junction"));
+        const dependencyLocation = dependencyNode.pkg.symLinkDir
+          ? path.resolve(dependencyNode.location, dependencyNode.pkg.symLinkDir)
```

Comment:
Wouldn't this be better served by the `lerna bootstrap` equivalent of [`--contents`](https://github.com/lerna/lerna/tree/master/commands/publish#--contents-dir)? This smells to high heaven of Angular-isms.